corehq/apps/case_importer/tests/test_importer.py

```diff
@@ -110,11 +111,11 @@ def testImportNamedColumns(self):
 
     @run_with_all_backends
     def testImportTrailingWhitespace(self):
-        cols = ['case_id', 'age', u'sex\xa0', 'location']
+        cols = ['case_id', 'age', 'sex\xa0', 'location']
```

Comment:
That's a hex character in a Unicode string. The reason this isn't breaking is because it corresponds to an [ASCII line feed character](https://www.cisco.com/c/en/us/td/docs/ios/12_4/cfg_fund/command/reference/cfnapph.html). The Unicode for line feed is `\u240a`.

Reply:
Are you saying that this test was implemented incorrectly, or that the updates here aren't right?

Reply:
Oh it's just a nit. The test is passing, so I'm fine with this. 

I think the test was implemented *one character* less than perfect: `u'sex\xa0'` should have been either `u'sex\u240a'` or `'sex\xa0'`.